trimmer_decorator.rb

```diff
@@ -0,0 +1,10 @@
+require './Decorator'
+
```

Comment:
- [ ] Please change `require './Decorator'` to `require './decorator'` to fix the code run error since `require './Decorator'` does not load the `decorator.rb` file. :pray: